dags/process_zip_dag.py

```diff
@@ -396,7 +437,8 @@ def send_post_request_to_reindex_search_service():
 add_missing_jpeg_extensions.set_downstream(strip_related_article_tags)
 strip_related_article_tags.set_downstream(strip_object_id_tags)
 strip_object_id_tags.set_downstream(add_missing_uri_schemes_task)
-add_missing_uri_schemes_task.set_downstream(wrap_article)
+add_missing_uri_schemes_task.set_downstream(send_article_info)
+send_article_info.set_downstream(wrap_article)
```

Comment:
not sure whether the article or its details should be sent first; the `content-store` and `scholarly-content-detail` are decoupled from each other so there is no strong dependency. Conceptually I would think the article should appear before being associated with a category, but it's not a strong requirement.